admin_guide/docker_registry.adoc

```diff
@@ -198,3 +242,4 @@ $ docker push 172.30.124.220:5000/openshift/busybox
 cf2616975b4a: Image successfully pushed
 Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
+====
```

Comment:
IWBN to add some guidance on what to do should those commands fail (or otherwise "not succeed").